Failure when starting up intero in the intero repo.

```
The following GHC options are incompatible with GHCi and have not been passed to it: -O2 -threaded
Configuring GHCi with the following packages: intero
Intero 0.1.20 (GHC 7.10.3)
Type :intro and press enter for an introduction of the standard commands.

/usr/bin/ld: /home/joe/src/haskell/intero/.stack-work/dist/x86_64-linux/Cabal-1.22.5.0/build/intero/intero-tmp/cbits/hschooks.o: relocation R_X86_64_32 against `.rodata.str1.8' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: final link failed: Nonrepresentable section on output
collect2: error: ld returned 1 exit status

```

This is on Archlinux.

--- a/intero.cabal
+++ b/intero.cabal
@@ -56,6 +56,8 @@ executable intero
     cbits/hschooks.c
   cpp-options:
     -DGHCI
+  cc-options:
+    -fPIC
   other-modules:
     InteractiveUI
     GhciMonad

